assets/v4.2.0/kube-apiserver/config.yaml

```diff
@@ -40,6 +40,8 @@ apiServerArguments:
   - application/vnd.kubernetes.protobuf
   advertise-address:
   - "{{ .ExternalAPIIPAddress }}"
+  cloud-provider:
```

Comment:
just want to toggle this through a string. I believe "" is default value so it should be that easy